Samples/Litium App demo/README.md

```diff
@@ -0,0 +1,45 @@
+# Litium App Demo
+
+**Tested in Litium version: Litium 8 beta**
+
+Introducing Litium App - a decoupled application that communicates with Litium using web API.
+
+This is the guide on how to develop an application as Litium App.
+
+## Instructions
+
+1. Install `Litium.AppManagement.Application` package to application.
+    ```
+    <ItemGroup>
+    <PackageReference Include="Litium.AppManagement.Application" Version="1.0.0-CI-*" />
```

Comment:
I think we should write something in the documentation that the projects should use the latest version.

We should also remove the `-CI-*` part because that is an internal prefix on the package before the package will be released and there use the `1.0.0` as version number instead.